### ISSUE TITLE
Deno runtime support

### ISSUE THREAD

**Issue by awnion** [tonlabs/ever-sdk-js#501]
Detail experiment can be found here: [readme](https://github.com/awnion/example-ever-sdk-deno#readme)

I think it's pretty easy to add Deno support. Deno can work almost like web runtime with wasm.
We just need to fix some hardcoded assumptions like `fetch('/evernode.wasm')` because this is too browser-specific.

1. Right now the naive approach doesn't work yet. Consider: 

```typescript
import { TonClient } from "npm:@eversdk/core";
import { libWeb } from "npm:@eversdk/lib-web"; // or lib-node

TonClient.useBinaryLibrary(libWeb);

const client = new TonClient();
const { phrase } = await client.crypto.mnemonic_from_random({});
console.log(phrase);
client.close();
```

Will give us: 

```log
error: Could not resolve 'npm:@eversdk/lib-web'
```

2. `lib-node` approach also doesn't work.

In short:
```log
error: Uncaught TypeError: /root/.tonlabs/binaries/1/eversdk.node: undefined symbol: SSL_CTX_free
```

Seems like something with dynamic linking made for nodejs runtime. It feels easily fixable since Deno is made on Rust and Eversdk too.

**Comment by elasticLove1** [comment]
Working on it now)

**Comment by awnion** [comment]
If I try like this
https://github.com/awnion/example-ever-sdk-deno/blob/main/sdk-deno-test.ts

Under docker image
https://github.com/awnion/example-ever-sdk-deno/blob/main/sdk-deno-test.Dockerfile

```Makefile
IMAGE_TAG := sdk-deno-test

run:
	docker build --progress=plain -t $(IMAGE_TAG) -f sdk-deno-test.Dockerfile .
	docker run --rm -ti -v $$(pwd):/workdir $(IMAGE_TAG) deno run -A --unstable "./sdk-deno-test.ts"
```

```bash
make run
```
Throws error:

```
>>> 1.44.1
error: Uncaught {
  code: 25,
  message: "Unknown function: ",
  data: { core_version: "1.44.1" }
}
make: *** [run] Error 1
```

So it partially works, might be a minor problem
